packages/teleport-project-generator/src/index.ts

```diff
@@ -332,6 +332,7 @@ export class ProjectGenerator implements ProjectGeneratorType {
           this.strategy?.resources?.path && {
             resources: {
               items: uidl?.resources?.items,
+              cache: uidl.resources.cache,
```

Comment:
if `uidl?.resources?.items,` needs `?` then `uidl.resources.cache` needs `?` or none of them do.

Reply:
We don't have `default` values for `items` So, the usual projects might not even have any resources in them. But we do have `default` values for `cache`. If there is no cache in the UIDL. By default the value is `60` so typescript is all happy anytime we read `uid.resources.cache`

Reply:
I was referring to the code, not application knowledge. If `uidl?` on first line, then `uidl?` on second line as well.